README.md

```diff
@@ -10,3 +10,21 @@
 
 1. [Programming Languages](Professional_Skills_0002/Programming_Languages_%232_0002.md)
 2. [Web Development](Professional_Skills_0002/Web_Development_%232_0002.md)
+
+## Naphtal 0003
+
+### General Skills
+
```

Comment:
Please, review and test your links they are not redirecting to the intended page.